openSUSE/open-build-service
# Update obs-build version to 20210120 release

The actual version of the `build` submodule (commit 52506fa4f4eabb1454ef697dd71d8f86022787ae, from 22nd August 2018) makes it not possible to run minitests in the development environment.

To reproduce the error:
- Checkout the `master` branch
- Run any minitest. (See instructions in the [developer documentation](https://github.com/openSUSE/open-build-service/wiki/Development-Environment-Tips-&-Tricks#run-minitest)).
- It will error, with something similar to:
```
Minitest::UnexpectedError:         RuntimeError: Backend died
            app/lib/backend/test.rb:17:in `block in start'
            app/lib/backend/test.rb:15:in `loop'
            app/lib/backend/test.rb:15:in `start'
            test/functional/request_controller_test.rb:10:in `setup'
            test/test_helper.rb:116:in `block in __run'
            test/test_helper.rb:116:in `map'
            test/test_helper.rb:116:in `__run'
```

To verify the fix:
- Checkout this branch
- Update the submodule: `git submodule update`
- Run any minitest. Any test will pass.

-----------------

If this PR requires any particular action or consideration before deployment,
please check the reasons or add your own to the list:


[1] https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org#when-there-are-migrations
[2] https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#you-want-real-people-to-test-your-feature
